RELEASE_NOTES.md

```diff
@@ -1,3 +1,72 @@
+
+### Foodcritic 6.1.1
+
+Foodcritic has been updated from 6.0.0 to 6.1.1 with new rules and support for the latest Chef:
+
+- Updated Chef Infra Client metadata for 15.1 to include the new chocolatey_feature resources as well as new properties in the launchd and chocolatey_source resources
+- Added new rule to detect large files shipped in a cookbook: FC123: Content of a cookbook file is larger than 1MB. Thanks [@mattray](http://github.com/mattray)
+- Allow configuring the size of the AST cache with a new --ast-cache-size command line option. Thanks [@Babar](http://github.com/Babar)
+
+### ChefSpec 7.4.0
+
+ChefSpec has been updated to 7.4 with better support stubbing commands and a new `policyfile_path` configuration option for specifying the path to the PolicyFile.
+
+### kitchen-dokken 2.7.0
+
+kitchen-dokken has been updated to 2.7.0 with new options for controlling how containers are setup and pulled. You can now disable user namespace mode when running privileged containers with a new `userns_host` config option. There is also a new option `pull_chef_image` (true/false) to control force pulling the chef image on each run to check for newer images. This now defaults to true so that testing on latest and current always actually mean latest and current. See the [kitchen-digitalocean readme](https://github.com/test-kitchen/kitchen-digitalocean/blob/master/README.md) for `kitchen.yml` config examples.
+
+### kitchen-digitalocean 0.10.4
+
+kitchen-digitalocean has been updated to 0.10.4 with support for new distros and additional configuration options for instance setup. You can now control the default DigitalOcean region systems are spun up in using a new `DIGITALOCEAN_REGION` env var. You can still modify the region in the driver section of your `kitchen.yml` file if you'd like, and the default region of `nyc1` has not changed. This release also adds slug support for `fedora-29`, `fedora-30`, and `ubuntu-19`. Finally if you'd like to monitor your test instances there is new `monitoring` configuration option in the `kitchen.yml` driver section that allows enabling DigitalOcean's instance monitoring. See the [kitchen-dokken readme](https://github.com/someara/kitchen-dokken/blob/master/README.md) for `kitchen.yml` config examples.
+
+### knife-vsphere 3.0.0
+
+knife-vsphere has been updated to 3.0. This new version adds support for specifying the `bootstrap_template` when creating new VMs. This release also improves how the plugin finds VM hosts in order to support hosts in nested directories.
+
+## End of Ubuntu 14.04 support
+
+Ubuntu 14.04 entered the end-of-life phase April 30, 2019. Since this version of Ubuntu is now end of life we have stopped building packages for Ubuntu 14.04. If you rely on Ubuntu 14.04 in your environment we highly recommend upgrading your host to Ubuntu 16.04 or 18.04.
```

Comment:
```suggestion
Ubuntu 14.04 entered the end-of-life phase April 30, 2019. Since this version of Ubuntu is now end-of-life, we have stopped building packages for Ubuntu 14.04. If you rely on Ubuntu 14.04 in your environment, we highly recommend upgrading your host to Ubuntu 16.04 or 18.04.
```